index.php

```diff
@@ -1,4 +1,8 @@
 <?php
 /**
  * Don't modify the files in this folder.
- */
\ No newline at end of file
+ *
+ * @package Adbusters
+ */
+
+// This comment makes phpcs happy.
```

Comment:
There's really no benefit to index.php files like these. It's not up to plugins to properly protect directory indexing on servers (it creates a false sense of "security"), even if serveers are configured to look for index.php, versus index.html or some other configured file name.

Best just to get rid of them where they are essentially just empty files, and avoid the need to jump through PHPCS hoops.